resources/features/Stratification.feature

```diff
@@ -0,0 +1,69 @@
+Feature: Create a Stratified Frame
+    This feature takes an input Frame that reflects some statistical unit type data assigned with a prn and stratifies it.
+    Stratas are defined in a provided properties file. For every given strata (without regard for the selection type) a
+    set of numerical range filters (e.g. payeEmployee, sic07) are imposed on the Frame. A produced stratified cell is
+    then allocated a cell number (as per strata) which is appended to each of its rows.
+
+    Scenario Outline: Apply sic07 and payeEmployee range filters on a Unstratified Frame
+        Given a Frame:
```

Comment:
As the test is to prove that units in a given frame can be allocated to a strata and a file output with a cell_no, I would restrict the input frame such that all units are allocated in the output.
A seperate test should deal with how units in the input frame are treated when they do not fall within scope of the Stratification Properties.

Reply:
Let's discuss this, not sure what the value of the first one test would be.